### DISCUSSION TITLE
Add ability to specify a table format for a relationship field

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5913] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
> We have configured the Query combo box for Agents to provide additional information from the Agent record e.g. Date of Birth. Whilst this is really helpful for working with Collectors, this additional information is also displayed for created by and modified by agents (along with any other Agents table QBX).
> 
>  
> 
> IS there a way to limit which tables are utilising configuration so that it is showing on a more limited (and selected) set of Agent QBXs?

**Describe the solution you'd like**
Users should be able to specify a table format to be used in the query combo box on the forms or in (formatted) view in queries. See the use case described above for why this should be possible.

**Reported By**
RBGE

**Comment by maxpatiiuk** [comment]
This is already possible

When you add query combo box on the form, you can specify a typesearch attribute
And in the typesearch definition in the XML you can say which formatter to use
By default it uses the default formatter

Tell me if you need a code sample
